src/matchers/toBeSealed/index.test.js

```diff
@@ -0,0 +1,11 @@
+import matcher from './';
+expect.extend(matcher);
+describe('.toBeSealed', () => {
```

Comment:
You seem to be missing the `'.not.toBeSealed'` test cases which is causing the coverage to drop and the build to break. If you add the tests for that it should all be ready to merge :smile: